styles/variables-silicon.less

```
@@ -176,39 +176,51 @@
 
 // Input
 // ---------------------------------------
-@agate-input-border-width: 3px;
+@agate-input-decorator-border-bottom-width: 3px;
 @agate-input-decorator-border-width: 0;
 @agate-input-decorator-focus-border-width: 0;
+@agate-input-decorator-has-value-opacity: 0.7;
 @agate-input-decorator-height: 60px;
+@agate-input-decorator-opacity: 0.4;
+@agate-input-disabled-opacity: 0.3;
 @agate-input-focused-opacity: 1;
-@agate-input-font-size: 28px;
-@agate-input-height: 33px;
-@agate-input-opacity: 0.5;
-@agate-input-icon-display: flex;
-@agate-input-icon-align: center;
-@agate-input-padding-top-bottom: 21px;
-@agate-input-padding-right: 33px;
-@agate-input-padding-left: 0;
-@agate-input-icon-after-padding-top-bottom: 21px;
-@agate-input-icon-after-padding-right: 21px;
+@agate-input-font-size: 30px;
+@agate-input-has-value-opacity: 0.6;
```

As i checked on GUI guide, it should be 0.5 not 0.6;
And this name is not proper.

hm..

```suggestion
@agate-input-icon-has-value-opacity: 0.6;
```

but I don't think it should be different with @agate-input-decorator-has-value-opacity;

---

I made the changes.